### PR TITLE
Update Inputfield, ChosenOne and Console.log

### PR DESCRIPTION
**Pull request by boldCM**
- change Inputfield to textarea tag
- use different regex to make input easier
- Add console.log feature...
- Fix escaping the lucky one 😜

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,6 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <script src="https://cdn.rawgit.com/adriancooney/console.image/c9e6d4fd/console.image.min.js"></script>
```
Comment (lmachens) [inline public/index.html]: > RawGit has reached the end of its useful life

Source: https://rawgit.com/

```suggestion
    <script src="https://raw.githubusercontent.com/adriancooney/console.image/master/console.image.min.js"></script>
```

File: src/App.js [hunk 2]
```diff
@@ -66,14 +66,20 @@ function App() {
             onClick={() => {
               if (values.length > 0) {
                 let randomPerson = choosOneRandom(values);
-                while (randomPerson.name === theLuckyHacker) {
-                  if (values.length > 0) {
-                    randomPerson = choosOneRandom(values);
-                    console.log("he got lucky :)");
-                  }
-                  break;
+                // while (randomPerson.name === theLuckyHacker) {
```
Comment (lmachens) [inline src/App.js]:
```suggestion
                while (randomPerson.name === theLuckyHacker) {
```

**Review by lmachens — COMMENTED** [review]
Can you add a description and revert uncomment lines?
Besides that, well done 👍.

File: public/index.html (hunk 1 above)
Comment (lmachens) [reply public/index.html]: Looks like raw.githubusercontent is blocking requests.
Alternative:
https://www.jsdelivr.com/?docs=gh
Usage:
https://cdn.jsdelivr.net/gh/adriancooney/console.image/console.image.min.js

**Comment by mhstrkmp** [comment]
🚀

**Review by bwnstck — APPROVED** [review]
lgtm 🚀

**Pull request merged**